### PR TITLE
Feature/prune

### PR DESCRIPTION
**Pull request by jpsl00**
- Add Prune
- Add PruneChannel
- Localize both commands

### CODE REVIEW

**Comment by jpsl00** [comment]
[Prune Command](https://trello.com/c/YwQcz7Me/3-prune-command)

**Comment by jpsl00** [comment]
[PruneChannel Command](https://trello.com/c/ErtpvCtL/9-prunechannel-command)

File: src/commands/Moderation/prune.ts [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+      aliases: ["purge", "clean"],
+      usage: "<Amount:number{1,50}> [Channel:channel]"
+    });
+
+    this.customizeResponse("Amount", msg =>
+      msg.language.get("COMMAND_PRUNE_INVALID")
+    );
+  }
+
+  public async run(msg: KlasaMessage, args): Promise<any> {
```
Comment (test137E29B) [inline src/commands/Moderation/prune.ts]: Please add typing here with `: [amountType, channelType]`

**Review by test137E29B — CHANGES_REQUESTED** [review]

File: src/commands/Moderation/prune.ts (hunk 1 above)
Comment (jpsl00) [reply src/commands/Moderation/prune.ts]: oki

**Review by test137E29B — APPROVED** [review]

**Pull request merged**